### ISSUE TITLE
Request auto position emojionearea-picker.

### ISSUE THREAD

**Issue by M-Husein** [mervick/emojionearea#131]
Hi, mervick

thanks for share your work ;)
I want Request auto position emojionearea-picker, like tether.js http://tether.io/.
I mean, the picker can adjust the available space position.

This is what i mean:
![illustrate](https://cloud.githubusercontent.com/assets/19644272/25783020/074fb29c-337f-11e7-80fb-9e3ac0c40c4f.png)

I hope you can add this feature for your awesome emojione plugin. 
Thanks you very much ;)

**Comment by mervick** [comment]
Hi,
Thank you for your suggestion,
I know about that bug.
I even already fixed this bug before, but only on in closed product with proprietary license and can't just share it.

**Comment by M-Husein** [comment]
Oh, I see :)

What available special feature in proprietary license?
Where I can show the list proprietary license?

**Comment by mervick** [comment]
No it used in proprietary product of company.

**Comment by M-Husein** [comment]
Oh, sorry I misunderstood :)

But, please give me little info or guide to implement this,
Do you use tether.js or build your own auto position system?

**Comment by mervick** [comment]
I build my own auto position system, it detect position of picker button and window height, also it reacts on window scrolling

**Comment by M-Husein** [comment]
Ok, I try build without tether.js, Thank you very much.

:+1:

**Comment by Zeokat** [comment]
This bug is really annoying. Could anyone solve it?

I love this project, but is useless on most mobile websites that i use it. The emojionearea can't be accessed properly.

Anyone knows an emoji picker without this issue?

**Comment by sharet08** [comment]
ojis. 😎🤠😏😭😡😰🤢

**Comment by sharet08** [comment]
😀🇵🇪

**Comment by M-Husein** [comment]
Hello all,
I change emojionearea picker with bootstrap 4 dropdown menu + popper.js, 
auto position success, @mervick saya pikir kamu harus pertimbangkan untuk memnggunakan popper.js.

Thanks :)

**Comment by mervick** [comment]
@M-Husein it will add additional dependency

**Comment by M-Husein** [comment]
@mervick, So, have you added this feature with your own code?

**Comment by mervick** [comment]
@M-Husein no, currently I didn't, but it's not good to add one more dependency for this.
Yes, popper.js is good, but as I remember there are a lot issues with proper version of popper.js and bootstrap 4.
I think popper.js can be used as plugin

**Comment by hacknlove** [comment]
> This bug is really annoying. Could anyone solve it?
> 
> I love this project, but is useless on most mobile websites that i use it. The emojionearea can't be accessed properly.
> 
> Anyone knows an emoji picker without this issue?

In mobile websites I hide the button, because users can add emojis from their virtual keyboads

```
    .emojionearea-button {
      display: none;
      @media (hover:hover) {
        display: block;
      }
    }
```

**Comment by hacknlove** [comment]
try this with "top" positioning

```
    events: {
      'picker.show' (picker) {
        if (picker.offset().top < 0) {
          picker.css('top', picker.position().top - picker.offset().top)
          picker.css('position', 'absolute')
        }
      },
      'picker.hide' (picker) {
        console.log('hide')
        picker.css('top', '')
        picker.css('position', '')
      }
    }
```
